spec/main.tex

```diff
@@ -886,13 +887,15 @@ \subsection{Provisioning Protocol}
 
       \item For $j \neq i$:
         \begin{itemize}
+          \item Assert $(N_j, s_j, t_j, \rho_j, u_j) \? \in
+            (\Z, \Z, \Z, \Bit^\kappa, \Bit^\kappa)$
```

Comment:
Surprisingly, we never require anywhere that $s_j < N_j$ and $t_j < N_j$. Probably it's redundant?

Reply:
Maybe because all operation are already modulo reduced? (I didn't check the fact I just stated)